OsmAnd-java/src/main/java/net/osmand/osm/OsmRouteType.java

```diff
@@ -0,0 +1,430 @@
+package net.osmand.osm;
+
+import net.osmand.binary.BinaryMapDataObject;
+import net.osmand.binary.BinaryMapIndexReader;
+import net.osmand.binary.BinaryMapRouteReaderAdapter;
+import net.osmand.binary.RouteDataObject;
+import net.osmand.router.network.NetworkRouteSelector;
+import net.osmand.util.Algorithms;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class OsmRouteType {
+	private static final List<OsmRouteType> values = new ArrayList<>();
+	public static final String DEFAULT_ICON = "special_marker";
+	public static final String DEFAULT_COLOR = "orange";
+
+	public static final String HIKING_ROUTES_OSMC_ATTR = "hikingRoutesOSMC";
```

Comment:
? should be it a field inside OsmRouteType or groupped somehow

Reply:
Move to the inner public interface RenderingPropertyConstants {